Directory.Build.targets.example

```diff
@@ -9,4 +9,13 @@
         </ReferencePath>
         <AssemblySearchPaths>$(AssemblySearchPaths);$(ReferencePath);</AssemblySearchPaths>
     </PropertyGroup>
+  <Target Name="PostBuild" AfterTargets="PostBuildEvent">
+    <!-- Copy to build folder -->
+    <Copy SourceFiles="$(TargetPath)" DestinationFolder="build/" />
+
+    <!-- Package after Release build -->
+    <Exec Condition="'$(OS)' == 'Windows_NT'" Command="powershell -executionpolicy bypass .\package.ps1 -NoArchive -OutputDirectory 'D:\Programs\Derail Valley\Mods'" />

```

Comment:
we can use `$(DvInstallDir)` here

Reply:
oh and btw this file has inconsistent indentation (2 spaces and 4)

Reply:
This should be fixed now